suites/quincy/rbd/tier-2_rbd_rest.yaml

```diff
@@ -0,0 +1,100 @@
+# Tier2: Usecase to cover RBD pool and image creation using REST API
+# Tier-level: 2
+# Test-Suite: tier-2_rbd_rest.yaml
+#
+# Ceph RBD component.
+#
+# Cluster Configuration:
+#    Node 4 must to be a client node
+tests:
+
+  # Setup the cluster
+  - test:
+      abort-on-fail: true
+      module: install_prereq.py
+      name: install ceph pre-requisites
+
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                mon-ip: node1
+                orphan-initial-daemons: true
+                skip-monitoring-stack: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: RHCS cluster deployment using cephadm
+      destroy-clster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        node: node4
+        install_packages:
+          - ceph-common
+          - fio
+        copy_admin_keyring: true
+      desc: Configure client node
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+
+  - test:
+      desc: Install rbd-nbd and remove any epel packages
+      module: exec.py
+      name: Install rbd-nbd
+      config:
+        sudo: true
+        commands:
+          - "rm -rf /etc/yum.repos.d/epel*"
+          - "dnf install rbd-nbd -y"
+
+  - test:
+      desc: Run rest API tests to create pool and image
+      module: test_rbd_rest_pool_image_creation.py
+      name: Run rest API tests to create pool and image
+      polarion-id: CEPH-XXXXX
+      config:
+        rep_pool_config:
+          rbd_pool:
+            rbd_image1_rest:
```

Comment:
take the config as 
_number_of_pools:
number_of_images:_

Reply:
no, it can be anything as we use in other yamls. If we take num_of_pools then they need to add workflow to create as many pools using REST endpoints